common/src/main/java/com/wynntils/overlays/objectives/GuildObjectiveOverlay.java

```diff
@@ -67,7 +67,20 @@ protected boolean isVisible() {
     public void render(
             GuiGraphics guiGraphics, MultiBufferSource bufferSource, DeltaTracker deltaTracker, Window window) {
         WynnObjective guildObjective = Models.Objectives.getGuildObjective();
+        if (guildObjective == null) return;
+        renderObjective(guiGraphics, bufferSource, guildObjective);
+    }
+
+    @Override
+    public void renderPreview(
+            GuiGraphics guiGraphics, MultiBufferSource bufferSource, DeltaTracker deltaTracker, Window window) {
+        WynnObjective guildObjective = Models.Objectives.getGuildObjective();
+        if (guildObjective == null) guildObjective = WynnObjective.DEMO_GUILD;
```

Comment:
Always use braces after if. The only exception are one-line returns (or break/continue).